.mailmap

```diff
@@ -360,6 +360,7 @@ Sébastien Han <shan@redhat.com> <seb@redhat.com>
 Sebastien Ponce <sebastien.ponce@cern.ch> <Sebastien.Ponce@cern.ch>
 Sebastien Ponce <sebastien.ponce@cern.ch> root <root@lxbre43a05.cern.ch>
 Sergey Arkhipov <nineseconds@yandex.ru>
+shangfufei <shangfufei@inspur.com>
```

Comment:
Is this a github username `shangfufei`? If so, please update your actual name, which is usually the Display Name in your email.